Scripts/gen_BeamToVD101.sh

```diff
@@ -70,8 +70,8 @@ for dirname in 000 001 002 003 004 005 006 007 008 009; do
 done
 
 # Cleanup
-echo "Removing produced files"
-rm -f beamToVD101Resampler.fcl
-rm -f EleBeamCat.txt
-rm -f MuBeamCat.txt
-echo "Finished"
+#echo "Removing produced files"
```

Comment:
Just checking: did you mean to leave this commented out?   Unless commented out code is debug code or code under development we would like it to be removed.

Reply:
Thanks for catching that Rob. That was used for debugging which is now resolved. I have pushed the change.